File: examples/sms/send/node/sendSms.js
import * as sms from "@m3o/services/sms";

// Send an SMS.
async function SendSms() {
  let smsService = new sms.SmsService(process.env.MICRO_API_TOKEN);
  let rsp = await smsService.send({
    from: "Alice",
    message: "Hi there!",
    to: "+447681129",
  });
  console.log(rsp);
}

await SendSms();

File: examples/sms/send/go/sendSms.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/sms"
)

// Send an SMS.
func SendSms() {
	smsService := sms.NewSmsService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := smsService.Send(&sms.SendRequest{
		From:    "Alice",
		Message: "Hi there!",
		To:      "+447681129",
	})
	fmt.Println(rsp, err)
}
